pkg/cluster/client.go

```diff
@@ -107,8 +110,11 @@ func NewClient(kubeConfigPath, contextStr string, options Options) (Client, erro
 		kubeConfigPath: kubeConfigPath,
 		currentContext: contextStr,
 	}
+	if options.RequestTimeout == 0 {
+		options.RequestTimeout = defaultTimeout
+	}
```

Comment:
Just asking: do we really prefer to do this?  The documentation of the field `restConfig.Timeout` used at line 227 of this file says that "a value of 0 means no timeout".  Why not leave that as a possibility in our own API? 

(If you decide to remove this, you no longer need a `defaultTimeout` constant above)

Reply:
Having no timeout is not great user experience, I would avoid doing `restConfig.Timeout = timeout` at line 227 if input timeout is 0, so that we will consider it default timeout which is 32 seconds